pkg/compass/schema/compass-definitions.yaml

```diff
@@ -51,3 +51,59 @@
       originalType: string
   requiredPrivileges:
     - VEHICLE_ALL_TIME_LOCATION
+- vspecName:  Vehicle.Powertrain.CombustionEngine.EOP
+  conversions:
+    - originalName: labels.engine.oil.pressure.value # In bar, so we need to convert to kPa
+      originalType: string
+      isArray: false
+  requiredPrivileges:
+    - VEHICLE_NON_LOCATION_DATA
```

Comment:
nit you don't need privileges in this doc they are all sourced from default-definitions.yaml

Reply:
oh, ok